Source/WebCore/page/FrameTree.cpp

```diff
@@ -215,7 +215,7 @@ Frame* FrameTree::child(unsigned index) const
 
 Frame* FrameTree::childByFrameID(FrameIdentifier frameID) const
 {
-    for (auto* child = firstChild(); child; child = child->tree().nextSibling()) {
+    for (auto* child = firstChild(); child; child = child->tree().traverseNext()) {
```

Comment:
This is incorrect for a function named "childByFrameID" but could be used in a function named "descendantByFrameID"